### PR TITLE
Print pricelist

### PR DESCRIPTION
**Pull request by gbhoparia**
Backend & UI:- Create price list template. 
1. Price List template should have logo center aligned 
2. Price List template should be able to open on a print tab on click of header link in the admin section 
3. Admin url link to be added in admin role

UI : Change Registration Page Fields
1. change Namn to Client Name
2. Address1 to Invoice Address 
3. Address2 to Delivery Address 
4. remove fax no field 
5. Org Name to change to Contact Person Name

Product List:
Price made round off to 2 decimal values

### CODE REVIEW

File: routes/printdoc.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var express = require('express');
+var router = express.Router();
```
Comment (mailamiton) [inline routes/printdoc.js]: is this file required .. if not pls remove it .. Thnaks

File: src/service/printServices.js [hunk 2]
```diff
@@ -3,38 +3,74 @@ var moment = require('moment');
 var fs = require('fs');
 var printer = require('printer');
 var appConst = require('../../utils/appConstants');
+var productServices = require('../service/productServices');
+var appConst = require('../../utils/appConstants');
 
 function getDynamicHtml(req){
 
       //Need to handle senarios for undefined . if we dont get data from function
```
Comment (mailamiton) [inline src/service/printServices.js]: pls remove commented .. part

File: src/service/printServices.js [hunk 3]
```diff
@@ -3,38 +3,74 @@ var moment = require('moment');
...
+            var data = {};
+             if(reqType === "ORDER_DETAILS"){
+                 data.orderDetails = req.body.orderDetails;
+                 data.orderComments = req.body.orderComments;
+                 data.username = req.session.userId;
+                     return resolve(data);
+             } else if(reqType === "PRICE_LIST"){
+                 productServices.getProducts().then(function(data) {
+                     data.products = JSON.parse(data[0].json);
+                     //data.status = req.query.status;
```
Comment (mailamiton) [inline src/service/printServices.js]: pl  remove

File: views/customer/print/index.ejs [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<!doctype html>
```
Comment (mailamiton) [inline views/customer/print/index.ejs]: is this file required? if not pls remove it ..

File: views/products/product-list.ejs [hunk 5]
```diff
@@ -25,14 +25,14 @@
             <div class="col-md-12" style="font-family: 'Century Bold'">Price List</div>
         </div>
         <div class="row form-group" >
-            <div class="col-md-10">
+            <div class="col-md-9">
```
Comment (mailamiton) [inline views/products/product-list.ejs]: pls make this same as earlier .ie 10,1,1 instead of 9,1,2

File: routes/printdoc.js (hunk 1 above)
Comment (gbhoparia) [reply routes/printdoc.js]: removed.

File: src/service/printServices.js (hunk 2 above)
Comment (gbhoparia) [reply src/service/printServices.js]: Done

File: src/service/printServices.js (hunk 3 above)
Comment (gbhoparia) [reply src/service/printServices.js]: Done.

File: views/customer/print/index.ejs (hunk 4 above)
Comment (gbhoparia) [reply views/customer/print/index.ejs]: Removed.

File: views/products/product-list.ejs (hunk 5 above)
Comment (gbhoparia) [reply views/products/product-list.ejs]: Done
